YoutubeExplode/Videos/VideoClient.cs

```diff
@@ -50,6 +49,7 @@ public async Task<Video> GetAsync(VideoId id)
                 id,
                 playerResponse.GetVideoTitle(),
                 playerResponse.GetVideoAuthor(),
+                playerResponse.GetVideoChannelId(),
```

Comment:
It seems this method is not part of the committed files. Did you forget it by accident?

Reply:
I think not because the [CI build](https://github.com/Tyrrrz/YoutubeExplode/pull/393/checks?check_run_id=762873644) ran successfully.

Reply:
Ahhh. My bad, it was already part of the code, sorry.